src/trace/span-inferrer.ts

```diff
@@ -523,4 +523,17 @@ export class SpanInferrer {
     };
     return new SpanWrapper(this.traceWrapper.startSpan("aws.eventbridge", options), spanWrapperOptions);
   }
+
+  getResourcePath(event: any): string {
+    const routeKey = event.requestContext.routeKey;
```

Comment:
Optional chaining so this can be called safely given the `any` type. Your checks on 529 are enough otherwise.

```suggestion
    const routeKey = event?.requestContext?.routeKey;
```